src/ModelHelpers/Attributes.php

```diff
@@ -746,6 +772,11 @@ public function fromDateTime($value)
 	 */
 	protected function asTimestamp($value)
 	{
+		if ($value == $this->db->getNullDate())
+		{
+			return -1;
```

Comment:
Why `-1`?

Reply:
because the null date `0000-00-00 00:00:00` is not a valid date.